var/spack/repos/builtin/packages/intel-mpi/package.py

```diff
@@ -83,7 +83,14 @@ def setup_dependent_package(self, module, dep_spec):
         else:
             raise RuntimeError('No suitable bindir found')
 
-        self.spec.mpicc = join_path(bindir, 'mpicc')
```

Comment:
The discussion was actually here https://github.com/LLNL/spack/pull/2980#issuecomment-276747987